tests/prague/eip7692_eof_v1/eip7069_extcall/test_gas.py

```diff
@@ -50,25 +57,29 @@ def gas_test(
 
     sender = pre.fund_eoa(10**18)
```

Comment:
```suggestion
    sender = pre.fund_eoa()
```
Can we take the opportunity to remove this hard-coded value?